spec/requests/permissions/permission_spec.rb

```diff
@@ -0,0 +1,32 @@
+
+describe 'Permission permissions' do
+  create_entire_hierarchy
+
+  given_the_route '/projects/{project_id}/permissions' do
+    {
+      id: reader_permission.id,
+      project_id: project.id
+    }
+  end
+
+  using_the_factory :permission, traits: [:owner], factory_args: -> { { project_id: project.id } }
+
+  for_lists_expects do |user, _action|
+    case user
+    when :admin, :owner
```

Comment:
editors cannot edit permissions? Or are they part of the owner role? Or just owner only

Reply:
Yeah only owners and admin can change permissions